docs/prepare-environment/custom-env.txt

```diff
@@ -59,96 +59,97 @@ It is also possible to set these variables for each accelerator type separately:
+orchestrates model training and evaluation in the container. Trial runner containers are also loaded
+with the experiment model definition and hyperparameter values for the current trial.
+
+GPU-specific versions of each library are automatically selected when running on agents with GPUs.
 
 .. _default-environment:
 
 Default Images
 ==============
 
-In the current version of Determined, experiments and tasks are executed in containers with the
-following:
-
--  Ubuntu 18.04
--  CUDA 11.1
--  Python 3.8.x
--  TensorFlow 2.4.x
--  PyTorch 1.9.x
-
-Determined will automatically select GPU-specific versions of each library when running on agents
-with GPUs.
-
-In addition to the above settings, all trial runner containers are launched with additional
-Determined-specific harness code that orchestrates model training and evaluation in the container.
-Trial runner containers are also loaded with the experiment's model definition and values of the
-hyperparameters for the current trial.
-
-.. note::
-
-   The default images are
-   ``determinedai/environments:cuda-11.3-pytorch-1.10-lightning-1.5-tf-2.8-gpu-0.17.15`` and
-   ``determinedai/environments:py-3.8-pytorch-1.10-lightning-1.5-tf-2.8-cpu-0.17.15`` for GPU and
-   CPU respectively.
++-------------+---------------------------------------------------------------------------------------+
+| Environment | File Name                                                                             |
++=============+=======================================================================================+
+| CPUs        | ``determinedai/environments:py-3.8-pytorch-1.10-lightning-1.5-tf-2.8-cpu-0.17.12``    |
++-------------+---------------------------------------------------------------------------------------+
+| Nvidia GPUs | ``determinedai/environments:cuda-11.3-pytorch-1.10-lightning-1.5-tf-2.8-gpu-0.17.12`` |
++-------------+---------------------------------------------------------------------------------------+
+| ROCm        | ``determinedai/environments:rocm-4.2-pytorch-1.9-tf-2.5-rocm-0.17.12``                |
++-------------+---------------------------------------------------------------------------------------+
 
-Older Images
-============
-
-Images that provide older versions of the frameworks are still available and supported. Note that
-the performance of some models can vary with different CUDA versions.
+Supported legacy Images
+=======================
 
 -  ``determinedai/environments:py-3.6.9-pytorch-1.4-tf-1.15-cpu-067db2b``
 -  ``determinedai/environments:py-3.6.9-pytorch-1.4-tf-2.2-cpu-067db2b``
 -  ``determinedai/environments:cuda-10.0-pytorch-1.4-tf-1.15-gpu-067db2b``
 -  ``determinedai/environments:cuda-10.1-pytorch-1.4-tf-2.2-gpu-067db2b``
 
+The performance of some models can vary with different CUDA versions.
+
+Container Contents
+==================
+
+Experiments and tasks are run in containers that include the following packages:
+
+-  CUDA 11.1
```

Comment:
default CUDA image currently has 11.3.

To be fair, this whole paragraph may not be super helpful: according to the previous section, tasks may run in CPU/CUDA/ROCm containers which would not have CUDA at all, or have ROCm.

Reply:
Will delete. Another unmaintainable item. If significant, these should be listed in a clear reference doc and not buried here.